src/components/UploadButton.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+const styles = {
```

Comment:
I don't understand why this style is not handle by external CSS. Not sure it's you're doing though but I'm still wondering, in case you know why…